configman/__init__.py

```diff
@@ -22,8 +22,12 @@
     regex_converter,
     timedelta_converter
 )
+
 from configman.environment import environment
-from configman.command_line import command_line
+# this next line brings in command_line and, if argparse is available,
+# a definition of the configman version of ArgumentParser
+from configman.commandline import *
```

Comment:
It just seems strange. It reminds me of the days when we used to say that configobj wasn't a required install and we did all sorts of back-bending to work around it and it usually go confusing. 

If you change commandline.py to something like this:
```python
except ImportError:
    import getopt as command_line
    ArgumentParser = None
```
we can avoid the star import.